packages/flutter_test/lib/src/finders.dart

```diff
@@ -677,6 +700,23 @@ class CommonSemanticsFinders {
   }
 }
 
+/// Provides lightweight syntax for getting frequently used text range finders.
+///
+/// This class is instantiated once, as [CommonFinders.textRange], under [find].
+final class CommonTextRangeFinders {
+  const CommonTextRangeFinders._();
+
+  /// Finds all occurrances of the given `substring` in the static text and
+  /// returns the [TextRange]s.
+  ///
+  /// If the `skipOffstage` argument is true (the default), then this skips
+  /// static text inside widgets that are [Offstage], or that are from inactive
+  /// [Route]s.
+  FinderBase<TextRangeContext> of(String substring, { bool skipOffstage = true }) {
```

Comment:
nit: `of` feels awfully generic as a finder name if the plan is to eventually add more TextRange finders here.